db/seeds.rb

```diff
@@ -376,6 +376,7 @@
       contact_via_connect: true,
       published: true,
       bumped_at: Time.current,
+      tag_list: Tag.order(Arel.sql("RANDOM()")).first(2).pluck(:name),
```

Comment:
I find it intersting that nowadays Rails encourages use of `Arel.sql` vs the old `Tags.order("random()")` 👍